pkg/artifactory/repositories.go

```diff
@@ -155,7 +155,7 @@ func mkRepoRead(pack PackFunc, construct Constructor) schema.ReadContextFunc {
 		resp, err := m.(*resty.Client).R().SetResult(repo).Get(repositoriesEndpoint + d.Id())
 
 		if err != nil {
-			if resp != nil && (resp.StatusCode() == http.StatusNotFound) {
+			if resp != nil && (resp.StatusCode() == http.StatusBadRequest || resp.StatusCode() == http.StatusNotFound) {
```

Comment:
it might be a little cleaner to use a case statement?

Reply:
How so?

Reply:
The `resp != nil` check makes it not as nice.